src/native-common/GestureView.tsx

```diff
@@ -207,13 +207,13 @@ export abstract class GestureView extends ViewBase<Types.GestureViewProps, {}> {
         }
 
         const initialTimeStamp = this._getEventTimestamp(this._lastGestureStartEvent);
-        const initialPageX = this._lastGestureStartEvent.pageX;
-        const initialPageY = this._lastGestureStartEvent.pageY;
+        const initialPageX = this._lastGestureStartEvent.pageX || 0;
+        const initialPageY = this._lastGestureStartEvent.pageY || 0;
 
         const timeStamp = this._getEventTimestamp(e);
 
         return (timeStamp - initialTimeStamp <= _tapDurationThreshold &&
-            this._calcDistance(initialPageX - e.pageX, initialPageY - e.pageY) <= _tapPixelThreshold);
+            this._calcDistance(initialPageX - (e.pageX || 0), initialPageY - (e.pageY || 0)) <= _tapPixelThreshold);
```

Comment:
For these types of events, we're guaranteed that pageX and pageY are defined. So you could keep the code as is or change it to e.pageX!!!, etc.